.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: monthly
+    time: "03:00"
```

Comment:
`time` came in automatically generated config already and it is in UTC-time format.